src/app/Plant/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Organism } from '../Organism';
+
+const PLANT_INITIATIVE = 0;
+export abstract class Plant extends Organism {
+  constructor(
+    imageSrc: string,
+    private readonly chanceOfSpreadingPerRound: number = 0.1,
```

Comment:
Maybe the default chance of spreading should be a constant as well to be easier to understand